actionview/lib/action_view/helpers/tags/select_renderer.rb

```diff
@@ -14,7 +14,7 @@ def select_content_tag(option_tags, options, html_options)
             add_default_name_and_id(html_options)
 
             if placeholder_required?(html_options)
-              raise ArgumentError, "include_blank cannot be false for a required field." if options[:include_blank] == false
+              raise ArgumentError, "include_blank cannot be false for the required field: #{html_options["name"]}" if options[:include_blank] == false
```

Comment:
Probably without the `:` character it would read more natural:
```suggestion
              raise ArgumentError, "include_blank cannot be false for the required field \"#{html_options["name"]}\"" if options[:include_blank] == false
```